scripts/v4-forge-doc.sh

```diff
@@ -0,0 +1,74 @@
+# Usage: ./scripts/v4-forge-doc.sh, called from the root directory
+
+# Find all the .md files in the source directory
+# @dev assumes that v4-periphery and v4-core are in the parent directory
+# i.e. /home/user/docs, /home/user/v4-periphery, /home/user/v4-core
+
+# Copy v4-core
+find ../v4-core/docs/src -type f -regex '.*\.sol/[^.]*\.[^.]*\.md' | while read file; do
```

Comment:
so requires v4-core to be in a sibling repo and updated to latest manually? Would prefer if you handle dep management locally via submodule or something